tests/robot/hardware/devices/camera/avigilon/snmp/storage.robot

```diff
@@ -0,0 +1,37 @@
+*** Settings ***
+Documentation       Hardware Camera Avigilon storage
+
+Resource            ${CURDIR}${/}..${/}..${/}..${/}..${/}..${/}..${/}resources/import.resource
+
+Test Timeout        120s
+
+
+*** Variables ***
+${CMD}      ${CENTREON_PLUGINS}
+...         --plugin=hardware::devices::camera::avigilon::snmp::plugin
+...         --mode=storage
+...         --hostname=127.0.0.1
+...         --snmp-port=2024
+
+
+*** Test Cases ***
+Avigilon camera Storage ${tc}/3
+    [Documentation]    Hardware Camera Avigilon Storage
+    [Tags]    hardware    avigilon    storage
+    ${command}    Catenate
+    ...    ${CMD}
+    ...    --snmp-community=hardware/devices/camera/avigilon/snmp/hardware-camera-avigilon
+    ...    --warning-status='${warning_status}'
+    ...    --critical-status='${critical_status}'
+
+    ${output}    Run    ${command}
+    ${output}    Strip String    ${output}
+    Should Be Equal As Strings
+    ...    ${output}
+    ...    ${expected_result}
+    ...    \nWrong output result for command:\n${command}\n\nExpected:\n${expected_result}\nCommand output:\n${output}\n\n
+
+    Examples:        tc    warning_status                               critical_status                             expected_result    --
+            ...      1     ${EMPTY}                                     ${EMPTY}                                    OK: state of the SD card: mediaNotPresent
+            ...      2     '\%{storage_state\} =~ /mediaNotPresent/'   ${EMPTY}                                    WARNING: state of the SD card: mediaNotPresent
+            ...      3     ${EMPTY}                                     '\%{storage_state\} =~ /mediaNotPresent/'  CRITICAL: state of the SD card: mediaNotPresent
```

Comment:
```suggestion
            ...      2     '${PERCENT}\{storage_state\} =~ /mediaNotPresent/'   ${EMPTY}                                    WARNING: state of the SD card: mediaNotPresent
            ...      3     ${EMPTY}                                     '${PERCENT}\{storage_state\} =~ /mediaNotPresent/'  CRITICAL: state of the SD card: mediaNotPresent
```

Reply:
Try with the ${PERCENT} builtin variable (not even sure the brackets need to be escaped)